src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,43 @@
+import { useParams } from 'react-router-dom';
+import { Tab } from '../../types/Tab';
+import { TabInfo } from '../TabInfo';
+
+type Props = {
+  tabs: Tab[]
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+}) => {
+  const { tabId = '' } = useParams();
+  const selectedTab = tabs.find(tab => tab.id === tabId)?.content;
+  const ifTabIdIsWrong = tabId === '' || selectedTab === undefined;
```

Comment:
You don't need this variable actually. And btw would be better to name it `isTabIdWrong`